src/dart/ast.rs

```diff
@@ -530,13 +780,21 @@ impl Default for OptionalArgKind {
     }
 }
 
+/// Formal function argument definition.
 #[derive(Debug)]
 pub struct ArgDef {
+    /// The metadata of the argument.
     pub meta: Meta,
+    /// Allows overriding the type with a subtype (`covariant T x`).
     pub covariant: bool,
+    /// The type of the argument.
     pub ty: VarType,
+    /// Qualifies the argument as field, used in constructors (`this.x`).
     pub field: bool,
+    /// Specifies if the default value is given through `=` instead of `:`.
+    /// `(int x = 5) instead of (int x: 5)`
```

Comment:
I think there should be more backticks, i.e.:
\`(int x = 5)\` instead of \`(int x: 5)\`